data/keynoters/laura-nolan.md

```diff
@@ -0,0 +1,14 @@
+---
+title:
+  Killer Robots Considered Harmful
+speaker: Laura Nolan
+bio: Laura Nolan is a software engineer who focuses on software operations and reliability. She holds degrees in Computer Science and in Ethics, and is currently completing another MA in Strategic Studies. Since 2018, Laura has been part of the Campaign to Stop Killer Robots, an umbrella organisation of NGOs that works towards a legally-binding instrument to regulate autonomous weapons.
+
+Affiliation: Campaign to Stop Killer Robots
```

Comment:
@patrick91 I cannot render this line. Probably because "affiliation" is taken as a key in some yaml file somewhere?

Reply:
maybe it is because it starts with an uppercase A? i'll take a proper look later

Reply:
done 😊